resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -597,6 +597,10 @@
                         "comment": "FIONBIO"
                     }
                 ]
+            },
+            {
+                "syscall": "recvmsg",
+                "comment": "Needed by micro-http to read from the byte stream."
```

Comment:
this should have gone before parameter-validated seccomp entries, and in that group should have been added based on alphabetical position of the syscall name

same for x86_64 list

This might look like a `nit` but it is the only _data structuring requirement_ for this file so it's actually important.